src/rastervision/rv_config.py

```diff
@@ -74,13 +77,13 @@ def set_tmp_dir(tmp_dir=None):
         # If directory cannot be made and/or cannot be interacted
         # with, fall back to default.
         except Exception as e:
-            print(
+            log.warning(
                 'Root temporary directory cannot be used: {}. Using root: {}'.
                 format(explicit_tmp_dir, RVConfig.DEFAULT_DIR))
             RVConfig.tmp_dir = RVConfig.DEFAULT_DIR
         finally:
             make_dir(RVConfig.tmp_dir)
-            print('Temporary directory is: {}'.format(RVConfig.tmp_dir))
+            log.info('Temporary directory is: {}'.format(RVConfig.tmp_dir))
```

Comment:
Can you make this a `log.debug`, so that it doesn't display unless you ask for verbose?